### PR TITLE
Add API

### PR DESCRIPTION
**Pull request by AlexiaArtemis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,63 @@
...
+
+Aquagraph visualization in arrows, add user input and use fastapi to make opperations in a Neo4j database.
+
+# Dependencies
+
+To connect with Neo4j and it's database, Neo4j must be installed locally:
+
+- Download Neo4j https://neo4j.com/download/
+
+Steps: Press Download. Add credentials. Click download desktop. Copy the activation key. Click and drag Neo4j Desktop to the Application folder. Open the Noe4j Desktop (go to Security & Privacy and click to open anyway). After you have open the Neo4j Desktop under the software registration paste the key that you have previously copied.
```
Comment (oddaspa) [inline README.md]: Typo: _Noe4j_

File: README.md [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
+1. Save the credentials `clarify-credentials.json` in the main folder
+2. Run the script `python process_items.py`
+
+- In the script you can define how many items to retrieve by changing the variable `n_items`, the number of created batches by changing the variable `n_batches` and the number of (random) initial connections between batch and items via the variable `connect_one_in_each`.
+
+3. Copy the content of the output file `result.json` to arrows.app by importing it.
+
+## How to run the API
+
+1. Make sure the Neo4j database is actived.
```
Comment (oddaspa) [inline README.md]: Typo: _activated_

File: README.md [hunk 3]
```diff
@@ -1,13 +1,63 @@
...
+2. cd to aquagraph_arrows
```
Comment (oddaspa) [inline README.md]: No folder named _aquagraph_arrows_

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+neo4j
+python-dotenv
+pyclarify
+pydantic
+fastapi
+uvicorn
```
Comment (oddaspa) [inline requirements.txt]: missing dependency _dotenv_

Comment (oddaspa) [reply requirements.txt]: using `python3 -m uvicorn main:app --port 8080 --reload` resolves missing dependency

File: README.md [hunk 5]
```diff
@@ -1,13 +1,63 @@
...
+3. run
+
+> uvicorn main:app --port 8080 --reload
+
+4. Go to http://localhost:8000/docs
```
Comment (oddaspa) [inline README.md]: Wrong port (use 8080 instead)

**Review by oddaspa — CHANGES_REQUESTED** [review]
Good stuff! I will add my part today. There are some typos in the readme. The file `precess_items_2.py` is a typo as well. I don't understand why we need two process_item files? What is the difference? There are also some debug prints in the code.

File: requirements.txt (hunk 4 above)
Comment (AlexiaArtemis) [reply requirements.txt]: added dotenv in requirements.txt

**Review by oddaspa — APPROVED** [review]

**Review by zehsilva — APPROVED** [review]
Good work!

**Pull request merged**